Assets/Scripts/Models/Furniture.cs

```diff
@@ -382,22 +477,39 @@ public string GetSpriteName()
         return ret.String;
     }
 
+    /// <summary>
+    /// Check if the position of the furniture is valid or not
```

Comment:
we should add that is called when placing the furniture. It makes no sense to call it once is is placed.